src/navigation/order/Order.tsx

```diff
@@ -1,23 +1,21 @@
-import { FlatList } from 'native-base';
-import { Text, View } from 'react-native';
-import { useNavigation } from '@react-navigation/native';
+import React, { useMemo, useState } from 'react';
+import { FlatList, View } from 'react-native';
 import { useSelector } from 'react-redux';
-import React, { useCallback, useMemo, useState } from 'react';
 
-import { getAspectRatio } from '../task/components/mapUtils';
-import { navigateToTask } from '../utils';
-import { selectTasksByOrder as selectTasksByOrderLogistics } from '../../redux/logistics/selectors';
 import { selectFilteredTasksByOrder as selectTasksByOrderCourier } from '../../redux/Courier/taskSelectors';
+import { selectTasksByOrder as selectTasksByOrderLogistics } from '../../redux/logistics/selectors';
+import { getAspectRatio } from '../task/components/mapUtils';
 
-import Details from '../task/components/Details';
-import i18n from '../../i18n';
-import TaskMiniMap from '../task/components/MiniMap';
-import OrderDetails from './components/OrderDetails';
 import { createCurrentTaskList } from '../../shared/src/logistics/redux/taskListUtils';
 import { blueColor } from '../../styles/common';
+import Task from '../../types/task';
+import TaskMiniMap from '../task/components/MiniMap';
+import OrderAccordeon from './components/OrderAccordeon';
+import OrderDetail from './components/OrderDetail';
+import { RouteType } from './types';
+import { Box } from '../../../components/ui/box';
```

Comment:
Why not use the "global" import like in the docs?

```
import { Box } from '@/components/ui/box';
```

The same applies to all other imports.